src/main/cms/cms_menu_osd.c

```diff
@@ -44,25 +44,35 @@
 
 #ifdef USE_EXTENDED_CMS_MENUS
 static uint16_t osdConfig_item_pos[OSD_ITEM_COUNT];
-
+static uint8_t osdConfig_one;
 static long menuOsdActiveElemsOnEnter(void)
 {
+    memcpy(&osdConfig_one, &osdConfig()->lq_format, sizeof(uint8_t));
```

Comment:
Codacy found an issue: [Does not check for buffer overflows when copying to destination (CWE-120).  Make sure destination can always hold the source data. ](https://app.codacy.com/gh/emuflight/EmuFlight/pullRequest?prid=6303053)